### PR TITLE
chore: bump framework

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

* Bumps frameworks to latest version

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History - `price-oracle`
 
+## 0.10.9 (2023-06-23)
+
+- Bumps `open-autonomy` framework to `v0.10.7`
```
Comment (Adamantios) [inline HISTORY.md:5]:
```suggestion
- Bumps `open-autonomy` framework to `v0.10.7` #73
```

File: HISTORY.md [hunk 2]
```diff
@@ -11,10 +15,10 @@
 - Bumps `open-autonomy` framework to `v0.10.5.post2` #70
 
 
-## 0.10.6 (2023-05-31)
+## 0.10.7 (2023-05-31)
 
 - Bumps to `tomte@v0.2.12` and cleans up the repo #69
-- Bumps `open-aea` and `open-autonomy` frameworks to `v1.34.0` and `v0.10.5.post1` correspondingly #68
+- Bumps `open-aea` and `open-autonomy` frameworks to `v1.35.0` and `v0.10.5.post1` correspondingly #68
```
Comment (Adamantios) [inline HISTORY.md:21]: Why this change?

Comment (dvilelaf) [reply HISTORY.md:21]: I bumped all the frameworks replicating what Ardian did on watchtower. Should I only bump open-autonomy?

Comment (Adamantios) [reply HISTORY.md:21]: No I mean you updated the history incorrectly. This is an older version of the oracle that you edited.

**Review by 0xArdi — APPROVED** [review]

File: HISTORY.md (hunk 2 above)
Comment (Adamantios) [reply HISTORY.md]: `0.10.7` needs to be `0.10.6` again as well :sweat_smile:

Comment (dvilelaf) [reply HISTORY.md]: done

**Review by Adamantios — APPROVED** [review]
